lib/ctblmono.gi

```diff
@@ -1013,6 +1015,8 @@ InstallMethod( IsMonomialNumber,
 ##
 #M  TestMonomialQuick( <chi> )  . . . . . . . . . . . . . . . for a character
 ##
+##  We may assume that <chi> is an irreducible character.
```

Comment:
"We may assume"? This makes it sound as if it was optional, or as if there was some underlying reason... Remove "may"?, resp. say "Assumes that..." ?

Or even clarify: "This function assumes that <chi> is an irreducible character, but does not verify this. Other inputs may thus leading to nonsensical output." ?

Reply:
O.k.